source/agora/common/Config.d

```diff
@@ -560,3 +574,79 @@ unittest
     auto scp_quorum = toSCPQuorumSet(quorum);
     assert(scp_quorum.toQuorumConfig() == quorum);
 }
+
+/*******************************************************************************
+
+    Parse the `event_handlers` config section
+
+    Params:
+        ptr = pointer to the Yaml node containing the loggers configuration
```

Comment:
It's not `loggers`. Please be careful when copy-pasting code.

Reply:
> It's not `loggers`. Please be careful when copy-pasting code.

I made a mistake.